src/test/java/pl/gov/coi/pomocua/ads/myoffers/MyOffersResourceTest.java

```diff
@@ -37,55 +44,93 @@ void tearDown() {
-        assertThat(offers).extracting("description").containsExactly(accOffer.description, materialAidOffer.description);
-    }
+            ResponseEntity<AccommodationOffer> response = getOffer(accOffer.id, AccommodationOffer.class);
 
-    @Test
-    void shouldReturnOffersOnlyForLoggedUser() {
-        UserId accommodationOfferUserId = new UserId("acommodation offer user id");
-        testUser.setCurrentUserWithId(accommodationOfferUserId);
-        AccommodationOffer accOffer = postOffer(AccommodationsTestDataGenerator.sampleOffer(), "accommodations", AccommodationOffer.class);
-
-        UserId materialAidOfferUserId = new UserId("job offer user id");
-        testUser.setCurrentUserWithId(materialAidOfferUserId);
-        postOffer(MaterialAidTestDataGenerator.sampleOffer(), "material-aid", MaterialAidOffer.class);
-
-        testUser.setCurrentUserWithId(accommodationOfferUserId);
-        BaseOffer[] offers = listOffers();
-        assertThat(offers).hasSize(1);
-        assertThat(offers[0]).isInstanceOf(AccommodationOffer.class);
-        assertThat(offers[0]).isEqualTo(accOffer);
-    }
+            assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
+            assertThat(response.getBody()).isEqualTo(accOffer);
+        }
 
-    @Test
-    void shouldReturnOfferForCurrentUser() {
-        testUser.setCurrentUserWithId(new UserId("accommodation offer user id"));
-        AccommodationOffer accOffer = postOffer(AccommodationsTestDataGenerator.sampleOffer(), "accommodations", AccommodationOffer.class);
+        @Test
+        void shouldReturnNotFoundWhenGettingOfferForOtherUser() {
+            testUser.setCurrentUserWithId(new UserId("accommodation offer user id"));
+            AccommodationOffer accOffer = postOffer(AccommodationsTestDataGenerator.sampleOffer(), "accommodations", AccommodationOffer.class);
 
-        ResponseEntity<AccommodationOffer> response = restTemplate.getForEntity("/api/secure/my-offers/" + accOffer.id, AccommodationOffer.class);
-        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
-        assertThat(response.getBody()).isEqualTo(accOffer);
-    }
+            testUser.setCurrentUserWithId(new UserId("different user id"));
+            ResponseEntity<AccommodationOffer> response = getOffer(accOffer.id, AccommodationOffer.class);
+
+            assertThat(response.getStatusCode()).isEqualTo(HttpStatus.NOT_FOUND);
+        }
 
-    @Test
-    void shouldReturnNotFoundWhenGettingOfferForOtherUser() {
-        testUser.setCurrentUserWithId(new UserId("accommodation offer user id"));
-        AccommodationOffer accOffer = postOffer(AccommodationsTestDataGenerator.sampleOffer(), "accommodations", AccommodationOffer.class);
+        @ParameterizedTest
+        @MethodSource("pl.gov.coi.pomocua.ads.myoffers.MyOffersResourceTest#differentOfferTypes")
+        void shouldNotReturnDeactivatedOffer(BaseOffer offer, String urlSuffix) {
```

Comment:
We don't want to return deactivated offers for owner? I thought that user might want to `recreate` it.